community_images/fluent-bit/bitnami/config/fluent-bit.config

```diff
@@ -0,0 +1,85 @@
+[SERVICE]
+    # Flush
+    # =====
+    # set an interval of seconds before to flush records to a destination
+    flush        1
+
+    # Daemon
```

Comment:
Please make sure we're exercising all the functionality that can be injected via conf files, majority content is commented here.